techniques/css/C39.html

```diff
@@ -2,102 +2,76 @@
-               but additional movement triggered by the scrolling can also trigger vestibular symptoms.
-            </p>
-            <p>The understanding document for <a href="../../understanding/motion-actuation.html#resources">Motion Actuation</a> includes links for changing the reduce motion setting.
-            </p>
+            <h2>解説</h2>
+            <p>この達成方法は 'prefers-reduced-motion' CSS メディアクエリを使用して、ユーザーが Web ページのアニメーションを非表示にできることを目的とする。</p>
+            <p>一部の利用者は、動きのあるコンテンツから、注意散漫又は吐き気を経験することがある。例えば、ページのスクロールによって要素が動く (スクロールに伴う必要不可欠な動きを除く) と、前庭障害が発生する可能性がある。これらのアニメーションを構成する CSS をメディアクエリで囲むことで、それらの症状を防ぐことができる。</p>
+            <p>これの代表的な例として、背景が前景とは異なる速度で動くパララックススクロールが挙げられる。スクロールによるページの動きは必要不可欠である（かつユーザーのコントロール下にある）が、スクロールによって発生する追加の動きは前庭障害を引き起こす可能性がある。</p>
+            <p><a href="../../understanding/motion-actuation.html#resources">動きによる起動</a>の理解に関するドキュメントにはモーション軽減設定の変更に関するリンクが含まれる。</p>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-'prefers-reduced-motion'-css-media-query">
-               <h3>Example 1: 'prefers-reduced-motion' CSS Media Query</h3>
-               <p>Users can indicate their motion preference for interfaces in their system and the
-                  'prefers-reduced-motion' CSS Media Query will respect that choice. CSS can then be
-                  applied to disable that motion for users that request it.
-               </p><pre>
+               <h3>事例 1: 'prefers-reduced-motion' CSS メディアクエリ</h3>
+               <p>ユーザーはシステム内のインタフェースのモーションに関する意向を設定することができ、'prefers-reduced-motion' CSS メディアクエリはその選択を尊重する。モーションに CSS を適用することで、要求したユーザーに対しては無効化することができる。</p>
+               <pre>
 @media (prefers-reduced-motion: reduce) {
   /* CSS to disable motion goes here */
 }</pre><p class="working-example">
-                  <a href="../../working-examples/css-reduced-motion-query/index.html">Working example of 'prefers-reduced-motion' CSS Media Query</a>
-                  
+                  <a href="../../working-examples/css-reduced-motion-query/index.html">'prefers-reduced-motion' CSS メディアクエリの実装例</a>
                </p>
             </section>
          </section>
          <section id="tests">
-            <h2>Tests</h2>
+            <h2>検証</h2>
             <section class="test-procedure">
-               <h3>Procedure</h3>
-               <p>For each interactive element that moves due to a user interaction:</p>
+               <h3>手順</h3>
+               <p>ユーザーのインタラクションによって動くインタラクティブな要素それぞれについて:</p>
                <ol>
-                  
-                  <li>Enable the 'Reduce Motion' setting in your system;</li>
-                  
-                  <li>Check that the motion is not essential;</li>
-                  
-                  <li>Check that the element does not move.</li>
-                  
+                  <li>システムの「視差効果を減らす」設定を有効化する;</li>
```

Comment:
セミコロンはあまり日本語では出てこないと思いますので

```suggestion
                  <li>システムの「視差効果を減らす」設定を有効化する、</li>
```